packages/kit/test/apps/basics/test/client.test.js

```diff
@@ -651,6 +651,21 @@ test.describe('Load', () => {
 			);
 		});
 	}
+
+	if (!process.env.DEV) {
+		test.skip('does not fetch __data.json if no server load function exists', async ({
```

Comment:
do you mean to always skip this test?

Reply:
yes, we can't unskip it until we implement #7967. doing that _and_ reconfiguring the Vite plugin would have been a lot for one PR